src/SelectableBox/SelectableBox.scss

```diff
@@ -0,0 +1,56 @@
+@import "variables";
+
+.pgn__selectable_box-set {
```

Comment:
Per the Figma spec:

> Scale down for mobile and stack boxes

Is there current support for the stacking of these boxes on smaller screen sizes? If not, what are you thoughts on adding that behavior?